ChineseFilter/sections/specific.txt

```diff
@@ -74,4 +74,8 @@ nfmovies.com##a[href][onclick^="openurl()"] img[src^="/static/"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 nfmovies.com##body div[id^="aaaDiv"]
 !+ PLATFORM(windows, mac, android)
-||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
\ No newline at end of file
+||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
+||cocomanhua.com/sw.js$script
+cocomanhua.com###ics
+cocomanhua.com##+js(addEventListener-defuser, click, popMagic)
+cocomanhua.com##+js(addEventListener-defuser, , popMagic)
```

Comment:
We use another syntax for scriptlets `#%#//scriptlet("prevent-addEventListener", "click", "popMagic")`.
Also this domain has already existed in the common rule (AdGuard Base).

Reply:
ok, marked the change